src/obu.rs

```diff
@@ -385,13 +385,10 @@ fn parse_seq_hdr(
         }
         hbd
     };
-    let monochrome;
-    if profile != Rav1dProfile::High {
-        monochrome = gb.get_bit() as u8;
-    } else {
-        // Default initialization.
-        monochrome = Default::default();
-    }
+    let monochrome = match profile {
+        Rav1dProfile::High => gb.get_bit() as u8,
+        _ => 0,
```

Comment:
```suggestion
        _ => Default::default(), // Default initialization.
```

Reply:
Why do you think this is better? I'm personally not a fan of `Default::default()` because it relies on contextual knowledge, so it is harder to review in a diff.

Reply:
Because in C, this variable is uninitialized in this case, and I think `Default::default()` (with the comment, too) that this is the default initialization, whatever that may be, and it doesn't matter what value it is.